containers.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+images="${PWD}/docker/images"
+
+log(){
+  echo "$(date +'%T') $1"
+}
+
+networks(){
+  docker network create \
+    todo-app
+}
+
+volumes(){
+  docker volume create \
+    todo-mysql-data
+}
+
+todo_app(){
+  docker run \
+    --detach \
+    --publish 3000:3000 \
+    --workdir "/app" \
+    --volume "${images}"/getting-started/app:/app \
```

Comment:
Not sure about the quoting here, it looks wonky.. would `--volume "${images}/getting-started/app:/app"` work?

Reply:
Also, not sure what the `getting-started/` folder is pointing at. It looks like a symlink in git?

Reply:
Fixed in [this](https://github.com/reload/onboarding-oliver/pull/1/commits/b0649acd7b692bf40b77cd82f14c35be5c03b51a) commit.

The `getting-started` folder is one of multiple 'projects'/images that I have comitted into /docker/images, but I'm explicitly using this in my script in this case. However, by doing it this way, the possibility to use our other images is still open.